python/looker_sdk/rtl/auth_session.py

```diff
@@ -95,10 +96,21 @@ def login_user(self, sudo_id: int) -> None:
                 self._login_user()
 
     def _login_admin(self) -> None:
+        config_data = self.settings.read_ini(
+            self.settings._filename, self.settings._section
+        )
+        client_id = os.getenv("LOOKER_CLIENT_ID") or config_data.get("client_id")
+        client_secret = os.getenv("LOOKER_CLIENT_SECRET") or config_data.get(
+            "client_secret"
+        )
+
+        if not (client_id or client_secret):
+            raise error.SDKError(f"Required auth credentials not found.")
```

Comment:
don't need an `f` string here

Reply:
I only left `LOOKER_CLIENT_ID` and `LOOKER_CLIENT_SECRET` to ensure that if the credentials are passed either through the config file or through env variables, `cattr.structure` works as expected and the credentials are not stored as attributes in the ApiSettings once instantiated

Reply:
ah, that makes sense